Update Stackable_Storage_Bins_Customizable.scad

dtswk opened the pull request:
Added in the ability to also make the front a pattern if you want the cutout.

dtswk committed:
Update Stackable_Storage_Bins_Customizable.scad

Added in the ability to also make the front a pattern if you want the cutout.

eloyasensio commented:
Thanks for your apportation, @dtswk 
I will check after holidays.

Happy Holidays!

dtswk committed:
Update Stackable_Storage_Bins_Customizable.scad

Changed the front solid part to be a percent more in keeping with the rest of the original authors code
